example/server/integrations.py

```diff
@@ -325,23 +325,24 @@ def scripts():
           gtag('js', new Date());
           gtag('config', 'UA-53373928-6');
         </script>
-        """,
+        """
+    ]
+    if "form" not in page_content and page_content.get("title") == _("Confirm Email"):
```

Comment:
This seems very fragile to have the string `Confirm Email` duplicated, if we change it above this will brack.  Can we use a constant or something?